projects/novo-elements/src/elements/form/extras/number-range/number-range.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, forwardRef, OnDestroy, OnInit } from '@angular/core';
+import { ControlValueAccessor, FormBuilder, FormGroup, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { NovoLabelService } from 'novo-elements/services';
+import { Helpers } from 'novo-elements/utils';
+import { Subject } from 'rxjs';
+import { filter, takeUntil } from 'rxjs/operators';
+
+@Component({
+  selector: 'novo-number-range',
+  template: `
+    <form [formGroup]="rangeForm">
+      <novo-flex justify='space-between' align='end' gap="1rem">
+        <novo-field mr="sm">
+          <input formControlName="min" novoInput type='number' [placeholder]="labels.minimumPlaceholder"/>
+        </novo-field>
+        <novo-field ml="sm">
+          <input formControlName="max" novoInput type='number' [placeholder]="labels.maximumPlaceholder"/>
+        </novo-field>
+      </novo-flex>
+      <novo-error *ngIf="rangeForm.hasError('minGreaterThanMax')" style="position: absolute">
+        {{ labels.minGreaterThanMax }}
+      </novo-error>
+    </form>
+  `,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => NumberRangeComponent),
+      multi: true
+    }
+  ]
+})
+export class NumberRangeComponent implements OnInit, OnDestroy, ControlValueAccessor {
+  rangeForm: FormGroup;
+  _onChange: (value: any) => void = () => { };
+  _onTouched = () => { };
+  private _destroyed = new Subject<void>();
+
+  constructor(public labels: NovoLabelService, private fb: FormBuilder) { }
```

Comment:
```suggestion
  constructor(public labels: NovoLabelService, private formBuilder: FormBuilder) { }
```

I usually prefer using the whole word for more clarity with variable names for readability